modules/setting/setting.go

```diff
@@ -1160,6 +1126,32 @@ func MakeManifestData(appName string, appURL string, absoluteAssetURL string) []
 	return bytes
 }
 
+// CreateOrAppendToCustomConf creates or updates the custom config.
+// Use the callback to set individual values.
+func CreateOrAppendToCustomConf(callback func(cfg *ini.File)) {
```

Comment:
Go generally isn't a fan of callbacks - but I guess this is cleaner and I'm not certain if there's a better way to do this.